rosidl_typesupport_c/src/type_support_dispatch.hpp

```diff
@@ -75,6 +76,7 @@ get_typesupport_handle_function(
       auto clib = static_cast<const Poco::SharedLibrary *>(map->data[i]);
       lib = const_cast<Poco::SharedLibrary *>(clib);
       if (!lib->hasSymbol(map->symbol_name[i])) {
+        fprintf(stderr, "Failed to find symbol '%s' in library\n", map->symbol_name[i]);
```

Comment:
Would it be useful to print library name too ?
`fprintf(stderr, "Failed to find symbol '%s' in library '%s'\n", map->symbol_name[i], library_name);`